### DISCUSSION TITLE
set function using multiple values

### DISCUSSION THREAD

**Discussion by arnoldsi-vii** [dynamoose/dynamoose#1213] [General]
First of all. I love dynamoose. It's really speed up our development using DynamoDb. Thank you for that.

I've been using [Set function](https://dynamoosejs.com/guide/Schema#set-function--async-function) to build our values with required prefixes and suffixes. The function only pass the value of specific property. is it possible to use other properties to build more complex value on the fly?

**Comment by fishcharlie** [comment]
@arnoldsi-vii Sadly that is not currently available. I'm planning on adding schema wide `set` and `get` functions in v3, but this functionality is not currently implemented.
